### PR TITLE
fix bug "saved config can broke UI"

### LINKED ISSUE #354: saved config canbroke the UI
If a saved config refer to a non existant charts, the Ui is broken : 

`Uncaught (in promise) TypeError: r.iconsUrl[n] is undefined`

### PR DESCRIPTION
**Pull request by alainds**
closes #354

### CODE REVIEW

File: src/core/usecases/restorablePackageConfigs.ts [hunk 1]
```diff
@@ -306,9 +306,10 @@ export const selectors = (() => {
                 const { packageName, catalogId } = restorablePackageConfig;
 
                 return {
-                    "logoUrl": !packageIcons.areFetched
-                        ? undefined
-                        : packageIcons.iconsUrl[catalogId][packageName],
+                    "logoUrl":
+                        !packageIcons.areFetched || !packageIcons.iconsUrl[catalogId]
+                            ? undefined
+                            : packageIcons.iconsUrl[catalogId][packageName],
```
Comment (garronej) [inline src/core/usecases/restorablePackageConfigs.ts:312]:
```suggestion
                    "logoUrl":
                        !packageIcons.areFetched
                            ? undefined
                            : packageIcons.iconsUrl[catalogId]?.[packageName],
```

**Review by garronej — COMMENTED** [review]
Thanks for the fix!

**Comment by garronej** [comment]
Thanks for the PR, we really appreciate your contribution.  
This patch does fix the symptoms but it doesn’t address the root cause.  
Would you make it so the button is disabled and maybe add a tooltip to explain why.  
(Also it shouldn't be configurable)  

Thanks in advance

**Comment by fcomte** [comment]
I think we should merge this fix. A new API version is coming. 
With this new version we will be able to know all the versions of all charts. This is a prerequisite to create a good feature about the lifecycle of saved config

**Comment by garronej** [comment]
@fcomte agreed

**Pull request merged**